app/src/main/java/com/vatsal/kesarwani/therapy/Activity/AddPost.java

```diff
@@ -78,12 +79,13 @@ protected void onCreate(Bundle savedInstanceState) {
         postLater.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if(postLater.isChecked())
-                {
+                if (postLater.isChecked()) {
```

Comment:
https://stackoverflow.com/a/11278528/12203379

Refer to this article to implement a listener instead of a variable to listen to the switch and modify UI accordingly.